jmh/jmh.bzl

```diff
@@ -77,7 +77,7 @@ def jmh_repositories(maven_servers = ["http://central.maven.org/maven2"]):
 
 def _scala_construct_runtime_classpath(deps):
     scala_targets = [d.scala for d in deps if hasattr(d, "scala")]
-    java_targets = [d.java for d in deps if hasattr(d, "java")]
+    java_targets = [d[JavaInfo] for d in deps if hasattr(d, "JavaInfo")]
```

Comment:
This was written before scala returned a java provider. I think we can remove the scala targets entirely and just use `targets` in fact we can probably combine this with the loop below that constructs the depset.

Reply:
Great, done!